CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - `filter_video_chat_ended`
   - `filter_video_chat_participants_invited`
   - `filter_web_app_data` 
+- Implement `GetChatId` for `teloxide_core::types::{BotCommandScope, Chat, ChatJoinRequest, ChatMemberUpdated, MessageCommon, Recipient, ResponseParameters, TargetMessage, UpdateKind}`.
```

Comment:
Use crates/teloxide-core/CHANGELOG.md

- Implementation of `GetChatId` for ... 
+ don't forget to add your pull request's number & link

Reply:
I think mentioning everything here makes more sense, since the trait is defined in `teloxide`. There are no changes `crates/teloxide-core`.